system/HTTP/RequestInterface.php

```diff
@@ -40,8 +40,8 @@ public function isValidIP(string $ip, ?string $which = null): bool;
      * Fetch an item from the $_SERVER array.
      * Supplied by RequestTrait.
      *
-     * @param string $index  Index for item to be fetched from $_SERVER
-     * @param null   $filter A filter name to be applied
+     * @param string          $index  Index for item to be fetched from $_SERVER
+     * @param int|string|null $filter A filter name to be applied
```

Comment:
Why `string`?

Reply:
I found the usage:

```
$request->getServer(['SERVER_PROTOCOL', 'REQUEST_URI']);
```

On find in code

Reply:
It it the first parameter as an array. The `$filter` is null.

Reply:
Ok, updated https://github.com/codeigniter4/CodeIgniter4/pull/7322/commits/d6c8e5a5eb96ae42c978ec5a9ab14b225c966742